### PR TITLE
Fix: Typos in docs home page

### PR DESCRIPTION
**Pull request by stoobie**
### Description of the Changes

Fix typos in docs home page.

### PR Preview URL

After you push a commit to this PR, a preview is built and a URL to the root of the preview appears in the comment feed.

Paste here the specific URL(s) of the content that this PR addresses.

### Check List

- [x] Changes have been done against main branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1251)

### CODE REVIEW

File: components/Starknet/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -1,10 +1,10 @@
 [id="overview"]
 = Overview
 
-Starknet is a permissionless, Validity-Rollup, also known as a zero-knowledge rollup (ZK rollup) for Ethereum. It operates as a Layer 2 (L2) blockchain, enabling any dApp to achieve massive scale for its computation without compromising on Ethereum’s composability
+Starknet is a permissionless Validity-Rollup, also known as a zero-knowledge rollup (ZK rollup) for Ethereum. It operates as a Layer 2 (L2) blockchain, enabling any dApp to achieve massive scale for its computation without compromising on Ethereum’s composability
```
Comment (stoobie) [inline components/Starknet/modules/ROOT/pages/index.adoc:4]:
```suggestion
Starknet is a permissionless Validity-Rollup, also known as a zero-knowledge rollup (ZK rollup) for Ethereum. It operates as a Layer 2 (L2) blockchain, enabling any dApp to achieve massive achieve massive computation scale without compromising on Ethereum’s composability
```

File: components/Starknet/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -1,10 +1,10 @@
 [id="overview"]
 = Overview
 
-Starknet is a permissionless, Validity-Rollup, also known as a zero-knowledge rollup (ZK rollup) for Ethereum. It operates as a Layer 2 (L2) blockchain, enabling any dApp to achieve massive scale for its computation without compromising on Ethereum’s composability
+Starknet is a permissionless Validity-Rollup, also known as a zero-knowledge rollup (ZK rollup) for Ethereum. It operates as a Layer 2 (L2) blockchain, enabling any dApp to achieve massive achieve massive computation scale without compromising on Ethereum’s composability
```
Comment (stoobie) [inline components/Starknet/modules/ROOT/pages/index.adoc:4]:
```suggestion
Starknet is a permissionless Validity-Rollup, also known as a zero-knowledge rollup (ZK rollup) for Ethereum. Starknet is a Layer 2 (L2) blockchain, enabling any dApp to achieve massive achieve massive computation scale without compromising on Ethereum’s composability
```

File: components/Starknet/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -1,10 +1,10 @@
 [id="overview"]
 = Overview
 
-Starknet is a permissionless, Validity-Rollup, also known as a zero-knowledge rollup (ZK rollup) for Ethereum. It operates as a Layer 2 (L2) blockchain, enabling any dApp to achieve massive scale for its computation without compromising on Ethereum’s composability
+Starknet is a permissionless Validity-Rollup, also known as a zero-knowledge rollup (ZK rollup) for Ethereum. Starknet is a Layer 2 (L2) blockchain, enabling any dApp to achieve massive achieve massive computation scale without compromising on Ethereum’s composability
```
Comment (stoobie) [inline components/Starknet/modules/ROOT/pages/index.adoc:4]:
```suggestion
Starknet is a permissionless Validity-Rollup, also known as a zero-knowledge rollup (ZK rollup) for Ethereum. As a Layer 2 (L2) blockchain, Starknet enables any dApp to achieve massive computation scale without compromising on Ethereum’s composability
```

**Review by ArielElp — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/1251#-:-NxCBI6r68v0yI4iF9_x:bnfp4nl)* status: all files reviewed, 3 unresolved discussions

**Pull request merged**